plugins/ConnectedLivingSpace/CLSAddon.cs

```diff
@@ -256,7 +264,6 @@ private void RebuildCLSVessel()
                         vessel.Clear();
                     }
                     this.vessel = null;
-                    this.selectedSpace = -1;

```

Comment:
selectedSpace was changed to WindowSelectedSpace to clearly delineate the difference in its use.  It is now strictly used for managing highlighting when the CLS Window is open.  With it decoupled from RebuildVessel, third parties can still alter highlighting, even when the window is open and a space is selected.  Works well in actual use with SM :)